plugin-server/src/main/pluginsServer.ts

```diff
@@ -393,6 +393,12 @@ export async function startPluginsServer(
                 'reset-available-features-cache': async (message) => {
                     await piscina?.broadcastTask({ task: 'resetAvailableFeaturesCache', args: JSON.parse(message) })
                 },
+                'populate-plugin-capabilities': async (message) => {
+                    // We need this to be done in only once
+                    if (hub?.capabilities.pluginScheduledTasks && piscina) {
```

Comment:
nit: I'd rename the `transpileFrontendApps` capability into `appManagementSingleton` and reuse it

Reply:
I was so confused ... the transpileFrontedApps isn't used anymore (Marius moved that code out), so definitely calls for renaming or removal